kowl/templates/deployment.yaml

```
@@ -79,19 +85,19 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: {{ include "kowl.secretName" . }}
-                  key: kafka-sasl-password
+                  key: {{ get .Values.secret.keyName "kafka-sasl-password" }}
             - name: KAFKA_TLS_PASSPHRASE
               valueFrom:
                secretKeyRef:
                  name: {{ include "kowl.secretName" . }}
-                 key: kafka-tls-passphrase
+                 key: {{ get .Values.secret.keyName "kafka-tls-passphrase" }}
             {{- if eq .Values.image.repository "quay.io/cloudhut/kowl-business" }}
             {{- if not .Values.kowl.config.cloudhut }}
             - name: CLOUDHUT_LICENSE_TOKEN
               valueFrom:
                 secretKeyRef:
                   name: {{ include "kowl.secretName" . }}
-                  key: cloudhut-license-token
+                  key: {{ get .Values.secret.keyName "cloudhut-license-token" }}
```

```suggestion
                  key: {{ get .Values.secret.keyName.cloudhut-license-token "cloudhut-license-token" }}
```

You forgot to add the actual keyname from the default variables for all the key names I think or am I missing something?

---

hypens are nor allowed supported as variable name in helm 3

---

Hey @jkroepke ,
sorry for getting back to your comment so late. I'm quite busy at the moment. My comment doesn't make sense at all.